extension-readme.md

```diff
@@ -1,182 +1,242 @@
 # <a id="top"></a>AWS Toolkit for Visual Studio Code
 
-The AWS Toolkit for Visual Studio Code is an extension that enables you to interact with [Amazon Web Services (AWS)](https://aws.amazon.com/what-is-aws/) from within Visual Studio Code.
+The *AWS Toolkit for Visual Studio Code* is an extension that enables you to interact with [Amazon Web Services (AWS)](https://aws.amazon.com/what-is-aws/) from within the Visual Studio Code editor.
 
-*This is pre-release software and we recommend against using it in a production environment.*
+The following screenshots show important parts of the Toolkit.
+
+### <a id="ui-components"></a>Fundamental UI Components
 
 ![Overview](./resources/marketplace/overview.png)
 
-## Contents
+### <a id="open-command-palette"></a>AWS Commands in the Command Palette
+
+![Command Palette](./resources/marketplace/open-command-palette.gif)
+___
+
+# <a id="contents"></a>Contents
 
 * [Features](#features)
-* [Additional Setup Steps](#additional-setup-steps)
+* [Setup](#additional-setup-steps)
+* [AWS Commands](#aws-commands)
 * [Usage](#usage)
 * [Get Help](#get-help)
 
-## <a id="features"></a>Features
+{ [Return to Top](#top) }
+___
 
-You can use the AWS Toolkit for Visual Studio Code to:
+# <a id="features"></a>Features
 
-* [Manage supported AWS resources in an AWS account](#managing-aws-resources). For example, you can:
-  
-  * List available AWS CloudFormation stacks that contain AWS serverless applications.
-  * List available AWS Lambda functions.
-  * Invoke available Lambda functions.
+You can use the AWS Toolkit for Visual Studio Code as follows:
 
-  ![AWS Explorer](./resources/marketplace/aws-explorer.png)
+* Develop serverless applications locally, and then deploy them to an AWS account (see [Usage](#usage))
+* Manage certain supported AWS resources in an AWS account (see [Usage](#usage)).
 
-* [Develop serverless applications locally, and then deploy them to an AWS account](#developing-serverless-applications).
+  This includes:
+  * Listing and deleting AWS CloudFormation stacks
+  * Listing and invoking AWS Lambda functions
+
+  For example:
+
+  ![AWS Explorer](./resources/marketplace/aws-explorer.png)
 
-[Return to Top](#top)
+{ [Return to Contents](#contents) } or { [Return to Top](#top) }
+___
 
-## <a id="additional-setup-steps"></a>Additional Setup Steps
+# <a id="additional-setup-steps"></a>Setup
 
-After you install the AWS Toolkit for Visual Studio Code, you must complete these additional steps to access most of its features:
+After you install the AWS Toolkit for Visual Studio Code, to access most of its features, you must complete the additional steps defined in the [Getting Started](https://docs.aws.amazon.com/console/toolkit-for-vscode/getting-started) topic of the _AWS Toolkit for Visual Studio Code User Guide_. These additional steps include the following:
```

Comment:
This section (and the following bullet points) feel a little cluttered. Maybe wait until the end to post some links? Not sure what the best path forward is...this could be the best path.

Reply:
I think you mean the "Setup" section.
Yes, I got the same feeling with the original, so I tried to emphasize the steps over the links.
If fact, with the optional SAM setup, I was able to include one link at the end, but not so much for the AWS setup.
We can examine it further to see if we can come up with something better.